dev-support/docker/Dockerfile_centos_7

```diff
@@ -38,8 +45,13 @@ RUN yum update -y \
     && yum groupinstall -y "Development Tools" \
     && yum install -y \
         centos-release-scl \
-        python3 \
-    && yum install -y $(pkg-resolver/resolve.py centos:7)
+        python3
+
+# Apply the script again because centos-release-scl creates new YUM repo files
+RUN pkg-resolver/set-vault-as-baseurl-centos.sh centos:7
+
+# hadolint ignore=DL3008,SC2046
```

Comment:
I readed our previous Dockerfiles and found that DL3008 and SC2046 were not ignored. Why we should ignore these two rules here?

Reply:
the hadolint rule codes can be found at https://github.com/hadolint/hadolint?tab=readme-ov-file#rules